migrations/script.py.mako

```diff
@@ -0,0 +1,22 @@
+"""${message}
```

Comment:
I did some cleanup of the template:

* Docstring now contains only the "message"
* Imports now come before the revision variables